egs/babel/s5d/run-4-anydecode.sh

```diff
@@ -507,18 +522,21 @@ if [ -f exp/$nnet3_model/final.mdl ]; then
   fi
   if [ ! -f $decode/.done ]; then
     mkdir -p $decode
+    ivec_param=
+    if [[ $nnet3_model !=  *noivec* ]] ; then
+        ivec_param="  --online-ivector-dir exp/nnet3/ivectors_${dataset_id} "
+    fi
     $decode_script --nj $my_nj --cmd "$decode_cmd" $rnn_opts \
           --beam $dnn_beam --lattice-beam $dnn_lat_beam \
-          --skip-scoring true  \
-          --online-ivector-dir exp/nnet3/ivectors_${dataset_id} \
+          $ivec_param --skip-scoring true  \
           exp/tri5/graph ${dataset_dir}_hires $decode | tee $decode/decode.log
 
     touch $decode/.done
   fi
 
   local/run_kws_stt_task2.sh --cer $cer --max-states $max_states \
     --skip-scoring $skip_scoring --extra-kws $extra_kws --wip $wip \
-    --cmd "$decode_cmd" --skip-kws $skip_kws --skip-stt $skip_stt  \
+    --cmd "$decode_cmd" --skip-kws $skip_kws --skip-stt $skip_stt --resolve-overlaps $resolve_overlaps \
```

Comment:
I can't see where in the BABEL setup you are calling any script with --resolve-overlaps set to true.  It seems to me that would make this an orphan option, and the general rule is that orphan options should be removed.